### ISSUE TITLE
Implementation inconsistency

### ISSUE THREAD

**Issue by fmocking** [drorlab/gvp-pytorch#2]
Hello, I was looking over the Tensorflow and Pytorch versions of the CPD model and realized they are inconsistent. 

In Tensorflow version when we are [converting input to the features](https://github.com/drorlab/gvp/blob/00323029899f11e1270f863d2847a9629a75936c/src/models.py#L96) there is an extra GVP layer. Here is the [node embedding](https://github.com/drorlab/gvp/blob/00323029899f11e1270f863d2847a9629a75936c/src/models.py#L442) and [edge embedding](https://github.com/drorlab/gvp/blob/00323029899f11e1270f863d2847a9629a75936c/src/models.py#L445) functions defined under [StructuralFeatures](https://github.com/drorlab/gvp/blob/00323029899f11e1270f863d2847a9629a75936c/src/models.py#L313) which is called [here](https://github.com/drorlab/gvp/blob/00323029899f11e1270f863d2847a9629a75936c/src/models.py#L96). Right after this we apply another set of GVP layers for both [edge](https://github.com/drorlab/gvp/blob/00323029899f11e1270f863d2847a9629a75936c/src/models.py#L98) and [vertex](https://github.com/drorlab/gvp/blob/00323029899f11e1270f863d2847a9629a75936c/src/models.py#L97) embeddings. 

Meanwhile in Pytorch version we are missing the GVP layers under StructuralFeatures. Instead edge and vertex features only normalize while converting the inputs to features under [ProteinGraphDataset](https://github.com/drorlab/gvp-pytorch/blob/main/gvp/data.py#L187). Then just like in Tensorflow version GVP layers applied to normalized [vertex](https://github.com/drorlab/gvp-pytorch/blob/main/gvp/models.py#L74) and [edge](https://github.com/drorlab/gvp-pytorch/blob/main/gvp/models.py#L75) features. 

Can you please clarify this issue?

**Comment by stgzr** [comment]
Same concern about the inconsistency.
